tests/integration/conftest.py

```diff
@@ -449,7 +440,7 @@ async def prometheus_fixture(
     model: Model, application_name: str
 ) -> typing.AsyncGenerator[Application, None]:
     """Deploy and yield prometheus charm application with relation to WordPress charm."""
-    prometheus = await model.deploy("prometheus-k8s", channel="stable", trust=True)
+    prometheus = await model.deploy("prometheus-k8s", channel="latest/edge", trust=True)
```

Comment:
Is this change to `latest/edge` temporary, like for Loki?
If so it might be good to add a comment, just like for Loki.

Reply:
Added comments for all the `edge` deployments. Thanks for the suggestion!